services/frontend-v3/src/components/resultsCard/ResultsCard.jsx

```diff
@@ -31,7 +31,10 @@ const ResultsCard = () => {
         `api/search/${type}?${queryString}&language=${locale}`
       );
 
-      setResults(result.data);
+      setTimeout(() => {
+        setResults(result.data);
+      }, 500);
+
```

Comment:
Should this be in the code?

Reply:
i left it for PR testing I will remove before merge

Reply:
removed